usr/src/test/zfs-tests/include/default.cfg

```diff
@@ -88,6 +88,11 @@ export TESTSUBFS=subfs
 export TESTSUBFS1=subfs1
 export TESTSUBFS2=subfs2
 
+# some test sub file system names
+export TESTSUBFS=subfs.$$
+export TESTSUBFS1=subfs1.$$
+export TESTSUBFS2=subfs2.$$
+
```

Comment:
Actually, looking at the diffs, the lines above this addition are the correct ones.